test/e2e/seeder/network-states/swap-state/withSwapContracts.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
:information_source:  this file will need to be expanded in the future, when more complex swaps are performed, as we'll need to modify several contracts and storages